Formula/noti.rb

```diff
@@ -17,19 +16,19 @@ class Noti < Formula
   depends_on "go" => :build
 
   def install
-    mkdir_p "#{buildpath}/src/github.com/variadico"
-    ln_s buildpath, "#{buildpath}/src/github.com/variadico/noti"
     ENV["GOPATH"] = buildpath
-    cd "#{buildpath}/src/github.com/variadico/noti" do
-      if build.stable?
-        system "go", "build", "-o", "#{bin}/noti"
-      else
-        system "go", "build", "-o", "#{bin}/noti", "cmd/noti/main.go", "cmd/noti/osx.go"
-      end
+
+    notipath = buildpath/"src/github.com/variadico/noti"
+    notipath.install Dir["{*}"]
```

Comment:
You should be able to get away with a more vanilla `notipath.install Dir["*"]` here I'd imagine.